src/plugins/overlay/overlay-en.hbs

```diff
@@ -356,7 +355,6 @@ wb.doc.on( "click vclick", "#overlay-open-btn", function( event ) {
 				</ol>
 			</li>
 		</ol>
-		<button class="btn btn-primary popup-modal-dismiss" type="button">Close modal popup</button>
```

Comment:
Can you add a blurb in the overlay/lighbox documentation on how a author can add his own custom button. May be just reuse this working example and specify that it's need to be in a modal-footer.